### PR TITLE
sticky pistons won't grab the wrong objects now

### PR DESCRIPTION
**Pull request by WilliamHarmonMS**
@joshlory @Hamms 

This is a fix for [issue#265](https://github.com/code-dot-org/craft/issues/265)

### CODE REVIEW

File: src/test-build-only/levels.js [hunk 1]
```diff
@@ -63,9 +63,9 @@ module.exports = Object.assign({
       "redstoneWire", "redstoneWire", "redstoneWire", "", "", "rails", "rails", "rails", "", "",
       "redstoneWire", "", "redstoneWire", "", "", "", "", "rails", "", "",
       "redstoneWire", "redstoneWire", "redstoneWire", "", "", "", "", "rails", "rails", "",
-      "", "railsRedstoneTorch", "", "", "", "", "rails", "", "rails", "",
+      "", "railsRedstoneTorch", "", "", "pistonDownSticky", "", "rails", "", "rails", "",
       "", "", "", "", "", "", "rails", "rails", "rails", "",
-      "", "", "", "", "", "", "", "", "", "",
+      "", "", "", "", "railsRedstoneTorch", "", "", "", "", "",
```
Comment (joshlory) [inline src/test-build-only/levels.js]: Is this missing the actual unit test?

Comment (WilliamHarmonMS) [reply src/test-build-only/levels.js]: oh bollocks... did I include a change to the level file? 😭

Comment (WilliamHarmonMS) [reply src/test-build-only/levels.js]: Bollocks... did I commit changes to levels? 😭

**Comment by Hamms** [comment]
The desire to eventually split up LevelBlock is at least half the reason I've been pushing so hard to put so much of the block logic in there!

**Pull request merged**